src/app/charts/line-graph.component.ts

```diff
@@ -131,22 +134,19 @@ export class LineGraphComponent {
     private drawTrendLine(): void {
         // Only draw if data and add trendline flag
         if (this.trendline && this.extractedData.length) {
-          // Set up if first time
-          if (!this.trendData) {
```

Comment:
Is there no need to check for `trendData`? (Unsure if the check might have been lost with the indentation change.)